src/api/goods.ts

```diff
@@ -1,11 +1,21 @@
 // eslint-disable-next-line
-const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
+import { Good } from "../Types/Good";
+const API_URL = 'https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json';
 
-export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
+export async function getAll(): Promise<Good[]> {
+  const response = fetch(API_URL);
+
+  return (await response).json();
```

Comment:
weird construction, you can use `then()` or put await on the right side of your response assignment